platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/internal/component/resolution/failure/ResolutionFailureHandlerIntegrationTest.groovy

```diff
@@ -1129,7 +1111,7 @@ class ResolutionFailureHandlerIntegrationTest extends AbstractIntegrationSpec {
             }
 
             val forceResolution by tasks.registering(ForceResolution::class) {
-                resolvedFiles.from(configurations.getByName("resolveMe"))
+                resolvedFiles.from(configurations.getByName("resolveMe").files)
```

Comment:
I don't think we want to do this. This calls the `getFiles` method on the configuration, which is a simple `Set<File>` and not a `FileCollection`. 

This changes causes the configuration to be realized at configuration time instead of being properly wired to the `resolvedFiles` property.